### PR TITLE
Story 12

### PR DESCRIPTION
**Pull request by amhursh**
Related Issue: #16 

Summary: Created order model. Implemented functionality for user to view all orders.

Author(s): amhursh

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -10,6 +10,8 @@
   post '/login', to: 'sessions#create'
   delete '/logout', to: 'sessions#destroy'
 
+  get '/orders', to: 'orders#index'
```
Comment (limsammy) [inline config/routes.rb:13]: Routes are looking very clean.

File: spec/features/user/user_can_view_past_orders_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe "User can see past orders" do
+	context "User visits /orders" do
+		scenario "User can see all orders" do
+			user = create(:user)
+			order1, order2, order3 = create_list(:order, 3, user: user)
```
Comment (limsammy) [inline spec/features/user/user_can_view_past_orders_spec.rb:7]: awesome method and variable assignment. looks good

**Review by limsammy — COMMENTED** [review]
Looks good to merge

File: spec/models/order_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (randy-darsh) [inline spec/models/order_spec.rb:4]: need to add order model tests

File: app/models/user.rb [hunk 4]
```diff
@@ -1,3 +1,4 @@
 class User < ApplicationRecord
   has_secure_password
+  has_many :orders
```
Comment (randy-darsh) [inline app/models/user.rb:3]: yes it does

**Pull request merged**